prime-router/metadata/schemas/non-standard/all-in-one-health-ca-covid-19.schema

```diff
@@ -129,14 +130,19 @@ elements:
       documentation: Translate inbound text to outbound SNOMED Codes
       csvFields: [{ name: Specimen Type, format: $display}]
 
+
+# Set fields needed for the LIVDlookup() Mapper to set the LOINC code and OBX.17 - Device Info
+    - name: equipment_model_name
+      csvFields: [{name: Device Identifier}]
+#      mapperOverridesValue: true
+
     - name: test_performed_code
       type: TEXT
       csvFields: [{name: Test Code}]
 
-    - name: equipment_model_name
-      type: TEXT
-      default: 99999999
-      csvFields: [{name: Device Identifier}]
+#    - name: test_kit_name_id
+#      type: TEXT
```

Comment:
Remove commented out lines

Reply:
Leaving comments for now to reference again if needed during testing in Staging.